### ISSUE TITLE
AttributeError: 'Include_constructor' object has no attribute 'stream'

### ISSUE THREAD

**Issue by Sdub76** [borgmatic-collective/docker-borgmatic#188]
The latest docker build on 12/12 broke all of my borgmatic instances on several machines in an identical way.  Reverting to the previous 1.7.5 docker image fixes it.

Noted that the only difference in the repo since the previous build is the bump from python 3.10.8-alpine3.17 to 3.11.1-alpine3.17. Perhaps a syntax difference in how included files are handled?

My borgmatic command is:

`borgmatic prune create -c /etc/borgmatic.d/local.yaml --files --stats --log-file-verbosity=1 --log-file /mnt/logs/local-backup.log`

[local.yaml](https://github.com/borgmatic-collective/docker-borgmatic/files/10221448/local.yaml.txt)
[common.yaml](https://github.com/borgmatic-collective/docker-borgmatic/files/10221447/common.yaml.txt)

Error is:
```
Traceback (most recent call last):
  File "/usr/local/bin/borgmatic", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/local/lib/python3.11/site-packages/borgmatic/commands/borgmatic.py", line 1023, in main
    configs, parse_logs = load_configurations(
                          ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/borgmatic/commands/borgmatic.py", line 793, in load_configurations
    configs[config_filename], parse_logs = validate.parse_configuration(
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/borgmatic/config/validate.py", line 99, in parse_configuration
    config = load.load_configuration(config_filename)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/borgmatic/config/load.py", line 33, in load_configuration
    return yaml.load(open(filename))
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/ruamel/yaml/main.py", line 434, in load
    return constructor.get_single_data()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/ruamel/yaml/constructor.py", line 121, in get_single_data
    return self.construct_document(node)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/ruamel/yaml/constructor.py", line 131, in construct_document
    for _dummy in generator:
  File "/usr/local/lib/python3.11/site-packages/ruamel/yaml/constructor.py", line 674, in construct_yaml_map
    value = self.construct_mapping(node)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/ruamel/yaml/constructor.py", line 444, in construct_mapping
    self.flatten_mapping(node)
  File "/usr/local/lib/python3.11/site-packages/borgmatic/config/load.py", line 210, in flatten_mapping
    included_value = representer.represent_data(self.construct_object(value_node))
                                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/ruamel/yaml/constructor.py", line 154, in construct_object
    data = self.construct_non_recursive_object(node)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/ruamel/yaml/constructor.py", line 189, in construct_non_recursive_object
    data = constructor(self, node)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/borgmatic/config/load.py", line 45, in include_configuration
    include_directories = [os.getcwd(), os.path.abspath(os.path.dirname(loader.stream.name))]
                                                                        ^^^^^^^^^^^^^
AttributeError: 'Include_constructor' object has no attribute 'stream'

```

**Comment by Sdub76** [comment]
Noted that the only difference in the repo since the previous build is the bump from python 3.10.8-alpine3.17 to 3.11.1-alpine3.17.  Perhaps a syntax difference in how included files are handled?

**Comment by grantbevis** [comment]
Fixed in #189

**Comment by witten** [comment]
The [main test suite](https://torsion.org/borgmatic/docs/how-to/develop-on-borgmatic/#automated-tests) of unit and integration tests does not require Postgres, but the end-to-end tests do. Let me know if I can help with setting up test running as part of your image building.

**Comment by witten** [comment]
Someone else is getting this same error (not with docker-borgmatic) using Python 3.10: https://projects.torsion.org/borgmatic-collective/borgmatic/issues/622

I'm not sure what's going on.

**Comment by witten** [comment]
Heads up that at least in one case, this is looking like a platform issue. It occurs on ARM64 but not AMD64. I'll post further updates on the borgmatic ticket.

**Comment by modem7** [comment]
This might give some insight: https://github.com/Homebrew/homebrew-core/pull/116510

**Comment by modem7** [comment]
Looks like the latest version of borgmatic may have resolved?

**Comment by witten** [comment]
That's correct! Released in borgmatic 1.7.6.
